pkgs/by-name/dr/draupnir/test-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix coreutils
+
+if [ ! -z "$CLEAN" ]; then
+  nix-env -e draupnir
+  nix-store --gc
+  nix-collect-garbage -d
```

Comment:
I don't like the way this script works. It mutates global state in funny ways which I think are not required? I think you can just use `nix-shell -I nixpkgs=./. -p draupnir` or so?

Either way I don't think it should be included.